File: spec/linked-list.spec.js
'use strict';

// Allow chai syntax like `expect(foo).to.be.ok;`
// jshint -W030

const chai = require('chai');
const chaiAsPromised = require('chai-as-promised');

chai.use(chaiAsPromised);

const expect = chai.expect;

const LinkedList = require('../lib/linked-list');

describe('LinkedList', function () {
  const list = new LinkedList();
  it('is a constructor', function () {
    expect(list instanceof LinkedList).to.be.true;
  });
});

File: spec/linked_list_spec.rb
# frozen_string_literal: true

require_relative '../lib/linked_list.rb'

RSpec.describe LinkedList do
  context 'initialized without arguments' do
    list = LinkedList.new
    it 'is of class LinkedList' do
      expect(list.class).to eq(LinkedList)
    end
  end
end
